compiler/testData/codegen/boxInline/smap/inlineOnly/stdlibInlineOnly.kt

```diff
@@ -0,0 +1,36 @@
+// IGNORE_BACKEND_MULTI_MODULE: JVM_IR
+// FILE: 1.kt
+package test
+
+@Suppress("INVISIBLE_MEMBER", "INVISIBLE_REFERENCE")
+@kotlin.internal.InlineOnly
+inline fun <T, R> T.myLet(block: (T) -> R) = block(this)
+
+// FILE: 2.kt
+import test.*
+
+fun box(): String {
+    // should not have a line number for 1.kt:7; *should* have one for the lambda
+    // in order to let the debugger break on it (KT-23064)
+    return "O".myLet { it + "K" }
```

Comment:
Please test also 
     return "O".myLet(fun (it: String) : String {return it +  "K"})